docs/how-tos/providing-images-for-img.rst

```diff
@@ -5,25 +5,18 @@ Providing images for ``<img>``
 
 :author: `pkel <https://github.com/pkel>`_
 
-In this how-to, you will learn how to create an endpoint for providing images to HTML :code:`<img>` tags without client side JavaScript.
-The resulting HTML might look like this:
+In this how-to, you will learn how to create an endpoint for providing images to HTML :code:`<img>` tags without client side JavaScript. In fact, the presented technique is suitable for providing not only images, but arbitrary files.
 
-.. code-block:: html
-
-   <img src="http://host/files/42/cats.jpeg" alt="Cute Kittens"/>
+We will start with a minimal example that highlights the general concept.
+Afterwards we present a more detailed solution that fixes a few shortcomings of the first approach.
 
-In fact, the presented technique is suitable for providing not only images, but arbitrary files.
+.. note::
 
-We will start with a minimal example that highlights the general concept.
-Afterwards we present are more detailed solution that fixes a few shortcomings of the first approach.
+   There are some caveats when saving images in the database as `mentioned in the PostgreSQL wiki <https://wiki.postgresql.org/wiki/BinaryFilesInDB>`_. Also, consider using a Content Delivery Network (CDN) for scalability purposes, see `this discussion <https://github.com/PostgREST/postgrest/issues/1077>`_ for reference.
```

Comment:
Can't think of good words now, but this needs to be more than "some caveats", almost like "think twice before doing this". Also, the CDN thing should be left out(I don't know how good it is to recommend it).